phoenix-core-server/src/main/java/org/apache/phoenix/coprocessor/PhoenixRegionServerEndpoint.java

```diff
@@ -61,7 +61,17 @@ public void validateLastDDLTimestamp(RpcController controller,
             RegionServerEndpointProtos.ValidateLastDDLTimestampRequest request,
             RpcCallback<RegionServerEndpointProtos.ValidateLastDDLTimestampResponse> done) {
         metricsSource.incrementValidateTimestampRequestCount();
-        ServerMetadataCache cache = getServerMetadataCache();
+        ServerMetadataCache cache;
+        try {
+            cache = getServerMetadataCache();
```

Comment:
With this feature, under what conditions can the creation of the server metadata cache fail? When this feature is enabled, do we need to take any server side action in the case of that failure? Should we block mutations in that case - if yes, how can we achieve that?